Mlem.xcodeproj/project.pbxproj

```diff
@@ -1477,6 +1484,7 @@
 				DEBUG_INFORMATION_FORMAT = "dwarf-with-dsym";
 				ENABLE_NS_ASSERTIONS = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
+				ENABLE_USER_SCRIPT_SANDBOXING = YES;
```

Comment:
There are additional lines like this, plus the version check

I don't know if they'll interfere badly with other people's setups, and this sounds like a security risk, could you remove them?

Reply:
Removed